debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9MemTagCommands.java

```diff
@@ -802,7 +802,7 @@ public long getAllocSize () {
 	}
 
 	/**
-	 * CustomComparetor is used to sort J9MemTags depending on their allocated sizes. 
+	 * CustomComparator is used to sort J9MemTags depending on their allocated sizes. 
```

Comment:
I think this, and the next, should be two words: Custom Comparator
Comparator should be capitalized as it refers to the interface used below.